packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -188,10 +188,7 @@ module.exports = function(log, config) {
       } catch (err) {
         if (err.statusCode === 404) {
           log.error('subhub.listSubscriptions.1', { uid, err });
-          // TODO: update with subhub listSubscriptions error response for invalid uid
-          if (err.message === 'invalid uid') {
-            throw error.unknownCustomer(uid);
-          }
+          throw error.unknownCustomer(uid);
```

Comment:
_Interestingly_, if the customer has no subscriptions, subhub doesn't return an empty list, but instead returns a 403 error ([spec](https://github.com/mozilla/subhub/blob/master/subhub/swagger.yaml#L82), [code](https://github.com/mozilla/subhub/blob/master/subhub/sub/payments.py#L194)). Worth handling now?

Reply:
Hmm, yeah maybe I should throw that in there - I hadn't seen that as an error case before now